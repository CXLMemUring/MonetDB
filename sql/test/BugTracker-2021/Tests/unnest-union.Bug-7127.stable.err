stderr of test 'unnest-union.Bug-7127` in directory 'sql/test/BugTracker-2021` itself:


# 14:49:37 >  
# 14:49:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_listenaddr=all" "--set" "mapi_port=36196" "--set" "mapi_usock=/var/tmp/mtest-239088/.s.monetdb.36196" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2021" "--set" "embedded_r=yes" "--set" "embedded_c=true"
# 14:49:37 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# builtin opt 	raw_strings = false
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_listenaddr = all
# cmdline opt 	mapi_port = 36196
# cmdline opt 	mapi_usock = /var/tmp/mtest-239088/.s.monetdb.36196
# cmdline opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2021
# cmdline opt 	embedded_r = yes
# cmdline opt 	embedded_c = true

# 14:49:37 >  
# 14:49:37 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-239088" "--port=36196"
# 14:49:37 >  


# 14:49:38 >  
# 14:49:38 >  "Done."
# 14:49:38 >  

