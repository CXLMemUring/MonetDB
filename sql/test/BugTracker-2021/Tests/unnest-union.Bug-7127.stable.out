stdout of test 'unnest-union.Bug-7127` in directory 'sql/test/BugTracker-2021` itself:


# 14:49:37 >  
# 14:49:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_listenaddr=all" "--set" "mapi_port=36196" "--set" "mapi_usock=/var/tmp/mtest-239088/.s.monetdb.36196" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2021" "--set" "embedded_r=yes" "--set" "embedded_c=true"
# 14:49:37 >  

# MonetDB 5 server v11.39.18 (hg id: af88b10dddb7)
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2021', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.343 GiB available main-memory of which we use 12.504 GiB
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2021 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://fedora:36196/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-239088/.s.monetdb.36196
# MonetDB/GIS module loaded
# MonetDB/R   module loaded
# MonetDB/SQL module loaded

# 14:49:37 >  
# 14:49:37 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-239088" "--port=36196"
# 14:49:37 >  

#start transaction;
#create table t_qh ( c_f INTEGER , c_y2 INTEGER , c_i768 INTEGER , c_tqx TEXT , c_mknkhml TEXT , primary key(c_f, c_y2), unique(c_y2) );
#insert into t_qh values (1,1,1,'a','a'), (2,2,2,'b','b'), (3,3,3,'c','c');
[ 3	]
#create table t_ckfystsc ( c_kvhq5p INTEGER , c_aifpl INTEGER , c_jf6 TEXT , c_f31ix TEXT NOT NULL, c_lk0zqfa INTEGER , c_qotzuxn INTEGER , c_w_z TEXT , primary key(c_lk0zqfa), unique(c_kvhq5p) );
#insert into t_ckfystsc values (1,1,'a','a',1,1,'a'), (2,2,'b','b',2,2,'b'), (3,3,'c','c',3,3,'c');
[ 3	]
#create table t_irwrntam7 as select ref_0.c_i768 as c0, ref_0.c_i768 as c1, ref_0.c_f as c2, ref_0.c_i768 as c3 from t_qh as ref_0 where 
#(ref_0.c_f in ( select ref_1.c_lk0zqfa as c0 from t_ckfystsc as ref_1)) and (ref_0.c_y2 >= ( select ref_0.c_y2 as c0 from t_qh as ref_7 union select ref_0.c_f as c0 from t_qh as ref_9));
#select ref_0.c_i768 as c0, ref_0.c_i768 as c1, ref_0.c_f as c2, ref_0.c_i768 as c3 from t_qh as ref_0 where 
#(ref_0.c_f in ( select ref_1.c_lk0zqfa as c0 from t_ckfystsc as ref_1)) and (ref_0.c_y2 >= ( select ref_0.c_y2 as c0 from t_qh as ref_7 union select ref_0.c_f as c0 from t_qh as ref_9));
% .,	.,	.,	. # table_name
% c0,	c1,	c2,	c3 # name
% int,	int,	int,	int # type
% 1,	1,	1,	1 # length
[ 1,	1,	1,	1	]
[ 2,	2,	2,	2	]
[ 3,	3,	3,	3	]
#select 1 from t_qh where c_f in (select 2 from t_ckfystsc) and c_y2 = (select c_y2 union select c_f);
% .%14 # table_name
% %14 # name
% tinyint # type
% 1 # length
[ 1	]
#rollback;

# 14:49:38 >  
# 14:49:38 >  "Done."
# 14:49:38 >  

